### PR TITLE
Remove uoa experiment

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/inbox/1125189844075764/1191127742724217/1191588970844276
Tech Design URL: 
CC: 

**Description**:
This PR removes the use our app experiment. It includes:
- Removes variants
- Doesn't register the event `UserEventKey.USE_OUR_APP_SHORTCUT_ADDED` after a shortcut to FB has been added to avoid triggering the fireproof dialog.
- Removes the notification worker to avoid showing the notification if it was already scheduled (caveat: app wasn't opened after updating)
- Deprecates the use our app tag to cancel any scheduled notifications.
- Doesn't schedule the notification anymore.
- Stops moving the users to established 3 days after installation.

**Steps to test this PR**:

**Normal Update**
1. Install the "first" apk and launch the app.
1. Kill the app
1. Install the "second" apk and launch the app.
1. Wait 20 seconds
1. You should not see the UOA notification and the app should not crash

**Normal Update Not Crashing**
1. Install the "first" apk and launch the app.
1. Kill the app
1. Install the "second" apk and DO NOT launch the app.
1. Wait 20 seconds
1. You should not see the UOA notification
1. The logcat should show an error because it couldn't find the worker but the app won't crash.

`Class not found: com.duckduckgo.app.notification.NotificationScheduler$UseOurAppNotificationWorker`

**Adding a shortcut doesn't register event**
1. Install the app from the branch and launch.
1. Go to facebook.com
1. Click on the menu and then add to home.
1. Add the shorcut.
1. In the app database check the table user_events.
1. The event `USE_OUR_APP_SHORTCUT_ADDED` should not exist.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutReceiver.kt [hunk 1]
```diff
@@ -56,9 +51,6 @@ class ShortcutReceiver @Inject constructor(
         GlobalScope.launch(dispatcher.io()) {
```
Comment (cmonfortep) [inline app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutReceiver.kt:51]: Consider removing the coroutine launch here, I think we don't need it anymore.

**Review by cmonfortep — COMMENTED** [review]
Just reviewed the code and looks great, good work @marcosholgado I've just commented on a minor thing.

On the testing side, could you clarify whats "Install the "first" apk"? and "second"? I've not found any apk attached to the implementation task. Thanks!

File: app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutReceiver.kt (hunk 1 above)
Comment (marcosholgado) [reply app/src/main/java/com/duckduckgo/app/browser/shortcut/ShortcutReceiver.kt]: Done!

**Comment by marcosholgado** [comment]
> Just reviewed the code and looks great, good work @marcosholgado I've just commented on a minor thing.
> 
> On the testing side, could you clarify whats "Install the "first" apk"? and "second"? I've not found any apk attached to the implementation task. Thanks!

Yup, apologies, no idea how I forgot to upload them. All the apks should be in that task now :)

**Review by cmonfortep — APPROVED** [review]
Tested and all works as described, thanks Marcos! 🚢 it

**Pull request merged**